Adapt to syncrhonous container operations

## Is your feature request related to a problem? Please describe.
I'm always frustrated when I see red in tests, like in https://rest.fs.neo.org/Aku38wpn2xEqfEPxmwyAa7N3VnAkMV8PEVkpLqwKM1z3/4464-1765181534/index.html#suites/570566fe391e15715e56e6453efdc0b9/4c5a48cc9b1825bb/ where ` tests.contract.test_n3.TestN3#test_n3_contract_witnesses_in_container_ops ` fails with 
> AssertionError: Regex pattern did not match.
 Regex: '.*container not saved.*'
 Input: 'Command: ./neofs-cli  request create-container /tmp/pytest-of-runner/pytest-0/popen-gw0/test_n3_contract_witnesses_in_0/create_container_request.json localhost:47531 \nError:\nreturn code: 1\noutput: Dialing localhost:47531...\nConnected. Sending request...\nError: transport failure: rpc error: code = DeadlineExceeded desc = context deadline exceeded\n'

## Describe the solution you'd like
Adapt test to node changes.

## Additional context
This is the consequence of https://github.com/nspcc-dev/neofs-node/pull/3711, the problem (request not served) happens earlier than before.

--- a/pytest_tests/tests/contract/test_n3.py
+++ b/pytest_tests/tests/contract/test_n3.py
@@ -204,7 +204,7 @@ class TestN3(TestNeofsBase):
                 invalid_container_invocation_script_b64,
                 owner_address,
             )
-            with pytest.raises(Exception, match=".*container not saved.*"):
+            with pytest.raises(Exception, match="(container not saved|DeadlineExceeded)"):
                 result = neofs_env.neofs_cli(None).request.create_container(
                     body=os.path.join(datadir, CONTAINER_REQUEST_JSON_FILE_NAME),
                     endpoint=neofs_env.storage_nodes[0].endpoint,
@@ -219,7 +219,7 @@ class TestN3(TestNeofsBase):
                 invalid_container_invocation_script_b64,
                 owner_address,
             )
-            with pytest.raises(Exception, match=".*container not saved.*"):
+            with pytest.raises(Exception, match="(container not saved|DeadlineExceeded)"):
                 result = neofs_env.neofs_cli(None).request.create_container(
                     body=os.path.join(datadir, CONTAINER_REQUEST_JSON_FILE_NAME),
                     endpoint=neofs_env.storage_nodes[0].endpoint,
@@ -236,7 +236,7 @@ class TestN3(TestNeofsBase):
                 invocation_script_wrong_signature_container_b64,
                 owner_address,
             )
-            with pytest.raises(Exception, match=".*container not saved.*"):
+            with pytest.raises(Exception, match="(container not saved|DeadlineExceeded)"):
                 result = neofs_env.neofs_cli(None).request.create_container(
                     body=os.path.join(datadir, CONTAINER_REQUEST_JSON_FILE_NAME),
                     endpoint=neofs_env.storage_nodes[0].endpoint,

